pages/instructorIssues/instructorIssues.ejs

```diff
@@ -29,112 +30,156 @@
-                  <a class="btn btn-primary" type="button" href="<%= urlPrefix %>/issues">
-                    <i class="fa fa-times" aria-hidden="true"></i>
-                    Clear
-                  </a>
-                </span>
+      <div class="card mb-4">
+        <div class="card-header bg-primary text-white">
+          <div class="d-flex flex-row align-items-center mb-2">
+            <div class="d-flex flex-column">
+              Issues
+              <small>
+                <a href="<%= commonQueries.allOpenQuery %>" class="mr-3 text-white">
+                  <i class="fa fa-exclamation-circle"></i> <%= openCount %> open
+                </a>
+                <a href="<%= commonQueries.allClosedQuery %>" class="text-white">
+                  <i class="fa fa-check-circle"></i> <%= closedCount %> closed
+                </a>
+              </small>
+            </div>
+            <button class="btn btn-sm btn-danger ml-auto" data-toggle="modal" data-target="#closeAllIssuesModal">
+              <i class="fa fa-times" aria-hidden="true"></i> Close all issues
+            </button>
+          </div>
+          <form name="query-form" method="GET">
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <button class="btn btn-med-light dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Filters</button>
+                <div class="dropdown-menu">
+                  <a class="dropdown-item" href="<%= commonQueries.allOpenQuery %>">Open issues</a>
+                  <a class="dropdown-item" href="<%= commonQueries.allClosedQuery %>">Closed issues</a>
+                  <a class="dropdown-item" href="<%= commonQueries.allManuallyReportedQuery %>">Manually-reported issues</a>
+                </div>
+              </div>
+              <input type="text" class="form-control" name="q" value="<%= filterQuery %>">
+              <div class="input-group-append">
+                <a class="btn btn-med-light" href="<%= urlPrefix %>/issues" title="Clear filters">
+                  <i class="fa fa-times" aria-hidden="true"></i>
+                </a>
+                <button class="btn btn-med-light" type="button" title="Show filter help" data-toggle="modal" data-target="#filterHelpModal">
+                  <i class="fa fa-question-circle" aria-hidden="true"></i>
+                </button>
               </div>
             </div>
+          </form>
+        </div>
+
+        <%
+        var getFormattedMessage = function(row) {
+          if (row.student_message == null || row.student_message.length == 0) {
+            return '—';
+          }
+
+          var msg = row.student_message;
+          if (row.manually_reported) {
+            msg = '"' + msg + '"';
+          }
+          return msg;
+        };
+        %>
+
+        <div class="list-group list-group-flush">
+          <% rows.forEach(row => { %>
```

Comment:
If we have zero rows, maybe we should display "no issues match the current filters" or something?

Reply:
Yep, that's a good idea.